{"name": "Login with valid phone number and invalid password -- @1.3 ", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//a[@data-csa-c-content-id='nav_ya_signin']\"}\n  (Session info: chrome=116.0.5845.111); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF7478A52A2+57122]\n\t(No symbol) [0x00007FF74781EA92]\n\t(No symbol) [0x00007FF7476EE3AB]\n\t(No symbol) [0x00007FF747727D3E]\n\t(No symbol) [0x00007FF747727E2C]\n\t(No symbol) [0x00007FF747760B67]\n\t(No symbol) [0x00007FF74774701F]\n\t(No symbol) [0x00007FF74775EB82]\n\t(No symbol) [0x00007FF747746DB3]\n\t(No symbol) [0x00007FF74771D2B1]\n\t(No symbol) [0x00007FF74771E494]\n\tGetHandleVerifier [0x00007FF747B4EF82+2849794]\n\tGetHandleVerifier [0x00007FF747BA1D24+3189156]\n\tGetHandleVerifier [0x00007FF747B9ACAF+3160367]\n\tGetHandleVerifier [0x00007FF747936D06+653702]\n\t(No symbol) [0x00007FF74782A208]\n\t(No symbol) [0x00007FF7478262C4]\n\t(No symbol) [0x00007FF7478263F6]\n\t(No symbol) [0x00007FF7478167A3]\n\tBaseThreadInitThunk [0x00007FFE43FE7614+20]\n\tRtlUserThreadStart [0x00007FFE45E826B1+33]\n\n", "trace": "  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login.py\", line 9, in step_impl\n    context.login_page = context.homepage.click_on_my_account()\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\home_page.py\", line 17, in click_on_my_account\n    self.click_on_element(\"my_account_option_xpath\", self.my_account_option_xpath)\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\BasePage.py\", line 32, in click_on_element\n    element = self.get_element(locator_type, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\BasePage.py\", line 46, in get_element\n    element = self.driver.find_element(By.XPATH, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 739, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "steps": [{"name": "Given I navigated to Login page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {\"method\":\"xpath\",\"selector\":\"//a[@data-csa-c-content-id='nav_ya_signin']\"}\n  (Session info: chrome=116.0.5845.111); For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception\nStacktrace:\n\tGetHandleVerifier [0x00007FF7478A52A2+57122]\n\t(No symbol) [0x00007FF74781EA92]\n\t(No symbol) [0x00007FF7476EE3AB]\n\t(No symbol) [0x00007FF747727D3E]\n\t(No symbol) [0x00007FF747727E2C]\n\t(No symbol) [0x00007FF747760B67]\n\t(No symbol) [0x00007FF74774701F]\n\t(No symbol) [0x00007FF74775EB82]\n\t(No symbol) [0x00007FF747746DB3]\n\t(No symbol) [0x00007FF74771D2B1]\n\t(No symbol) [0x00007FF74771E494]\n\tGetHandleVerifier [0x00007FF747B4EF82+2849794]\n\tGetHandleVerifier [0x00007FF747BA1D24+3189156]\n\tGetHandleVerifier [0x00007FF747B9ACAF+3160367]\n\tGetHandleVerifier [0x00007FF747936D06+653702]\n\t(No symbol) [0x00007FF74782A208]\n\t(No symbol) [0x00007FF7478262C4]\n\t(No symbol) [0x00007FF7478263F6]\n\t(No symbol) [0x00007FF7478167A3]\n\tBaseThreadInitThunk [0x00007FFE43FE7614+20]\n\tRtlUserThreadStart [0x00007FFE45E826B1+33]\n\n", "trace": "  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\login.py\", line 9, in step_impl\n    context.login_page = context.homepage.click_on_my_account()\n                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\home_page.py\", line 17, in click_on_my_account\n    self.click_on_element(\"my_account_option_xpath\", self.my_account_option_xpath)\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\BasePage.py\", line 32, in click_on_element\n    element = self.get_element(locator_type, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\PycharmProjects\\pythonProject\\POC-\\features\\pages\\BasePage.py\", line 46, in get_element\n    element = self.driver.find_element(By.XPATH, locator_value)\n              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 739, in find_element\n    return self.execute(Command.FIND_ELEMENT, {\"using\": by, \"value\": value})[\"value\"]\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\lmallikarjuna\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "attachments": [{"name": "screenshot", "source": "2e48dcb9-1f90-43b8-bc28-a87b789ba01b-attachment.png", "type": "image/png"}], "start": 1693385562347, "stop": 1693385562575}, {"name": "When I enter valid phone number as \"8019713239\" and invalid password as \"Harish@AMAZON33\" into the fields", "status": "skipped", "start": 1693385565210, "stop": 1693385565211}, {"name": "And I click on Login button", "status": "skipped", "start": 1693385565211, "stop": 1693385565211}, {"name": "Then I should get a proper warning message on password page", "status": "skipped", "start": 1693385565211, "stop": 1693385565211}], "parameters": [{"name": "number", "value": "8019713239"}, {"name": "password", "value": "Harish@AMAZON33"}], "start": 1693385552886, "stop": 1693385565211, "uuid": "bec877e4-dafc-41ee-8782-d41f63e2f1fc", "historyId": "9579b61a38b537dcda449bd87b6fa29d", "fullName": "Login Functionality: Login with valid phone number and invalid password", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "login"}, {"name": "feature", "value": "Login Functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}